src/app.js

```diff
@@ -61,7 +83,14 @@ class Messenger extends EventEmitter {
       this.cache = new Cacheman('sessions', {ttl: SESSION_TIMEOUT_MS / 1000});
     }
 
-    this.pages = pages;
+    if (pages && Object.keys(pages).length) {
+      this.pages = pages;
+    } else if (config.has('messenger.pageAccessToken') && config.has('facebook.pageId')) {
+      this.pages = {[config.get('facebook.pageId')]: config.get('messenger.pageAccessToken')};
+    } else {
+      this.pages = {};
+      debug("MISSING options.pages; you won't be able to reply or get profile information");
```

Comment:
This is my favorite line in this PR. It shows how those two config variables are actually optional depending on what your bot is doing. While the SDK doesn't exactly gracefully degrade, we could make it.